ivadomed/testing.py

```diff
@@ -236,11 +224,17 @@ def run_inference(test_loader, model, model_params, testing_params, ofolder, cud
                     preds_npy_list.append(output_nii.get_fdata().transpose(3, 0, 1, 2))
                     gt_lst = []
                     for gt in metadata[0]['gt_filenames']:
-                        gt_lst.append(nib.load(gt).get_fdata())
+                        # For multi-label, if all labels are not in every image
+                        if gt is not None:
+                            gt_lst.append(nib.load(gt).get_fdata())
+                        else:
```

Comment:
Could be done in another PR: but I think we should have a convention re this. What do you think @andreanne-lemay ?